kivymd/uix/button/button.py

```diff
@@ -904,14 +908,16 @@ def on_icon_color(self, instance_button, color: list) -> NoReturn:
         If we are setting an icon color, set theme_icon_color to Custom.
         For backwards compatibility (before theme_icon_color existed).
         """
-        if color and (self.theme_text_color == 'Custom'):
+
+        if color and (self.theme_text_color == "Custom"):
             self.theme_icon_color = "Custom"
 
 
-class ButtonContentsText():
+class ButtonContentsText:
     """
     Contents for BaseButton consisting of a single label.
     """
+
     pass
```

Comment:
It is not necessary to specify the `pass` instruction

Reply:
Hadn't realised (although I should have done) that the docstring is sufficient.... pass statement removed.

Reply:
Hadn't realised that the docstring was sufficient (although I should have done)... pass statement removed.